http://bugs.gentoo.org/289756
http://thread.gmane.org/gmane.comp.gnu.binutils/44450

revision 1.5
date: 2009/10/23 05:45:27;  author: amodra;  state: Exp;  lines: +5 -0
    * plugin.c: Produce empty object if not BFD_SUPPORTS_PLUGINS.

Index: bfd/plugin.c
===================================================================
RCS file: /cvs/src/src/bfd/plugin.c,v
retrieving revision 1.4
retrieving revision 1.5
diff -u -p -r1.4 -r1.5
--- bfd/plugin.c	29 Aug 2009 22:10:58 -0000	1.4
+++ bfd/plugin.c	23 Oct 2009 05:45:27 -0000	1.5
@@ -20,6 +20,10 @@
    MA 02110-1301, USA.  */
 
 #include "config.h"
+#include "bfd.h"
+
+#if BFD_SUPPORTS_PLUGINS
+
 #include <assert.h>
 #include <dlfcn.h>
 #include <stdarg.h>
@@ -492,3 +496,4 @@ const bfd_target plugin_vec =
 
   NULL  			/* backend_data.  */
 };
+#endif /* BFD_SUPPORTS_PLUGIN */
